src/cache_cf.cc

```diff
@@ -784,7 +784,7 @@ configDoConfigure(void)
      * the extra space is for loop detection in client_side.c -- we search
      * for substrings in the Via header.
      */
-    snprintf(ThisCache2, sizeof(ThisCache), " %s (%s)",
+    snprintf(ThisCache2, sizeof(ThisCache2), " %s (%s)",
```

Comment:
> Francesco: Would you mind updating the PR description with more context?

@kinkie, to be honest, I think the diff speaks for itself: We were filling ThisCache2 buffer using ThisCache size. I polished the PR title. Does the new title make the change clear enough?

FTR, the two buffers have the same size right now:

```C++
char  ThisCache[RFC2181_MAXHOSTNAMELEN << 1];
char ThisCache2[RFC2181_MAXHOSTNAMELEN << 1];
```